app/src/main/res/values/strings.xml

```diff
@@ -64,4 +64,13 @@
     <string name="verified_your_email_address">verified your email address?</string>
     <string name="verified">Verified</string>
     <string name="verified_your_email_address_click_below">verified your email address? click below!</string>
+    <string name="reset_pass_text"><u>Forgot password?</u></string>
+    <string name="reset_pass_msg">Enter the email for your Gathering\'s account and we\'ll send you a link to reset your password.</string>
+    <string name="forgot_password">Forgot password?</string>
+    <string name="tv1">You should receive your reset link soon.</string>
```

Comment:
we need beterr names for these
this could be named something like: reset_pass_email_sent_message